test/e2e_node/services/kubelet.go

```diff
@@ -165,14 +165,18 @@ func (e *E2EServices) startKubelet() (*server, error) {
 		// - test/e2e_node/conformance/run_test.sh.
 		"--pod-cidr", "10.100.0.0/24",
 		"--eviction-pressure-transition-period", "30s",
-		// Apply test framework feature gates by default. This could also be overridden
-		// by kubelet-flags.
-		"--feature-gates", framework.TestContext.FeatureGates,
 		"--eviction-hard", "memory.available<250Mi,nodefs.available<10%,nodefs.inodesFree<5%", // The hard eviction thresholds.
 		"--eviction-minimum-reclaim", "nodefs.available=5%,nodefs.inodesFree=5%", // The minimum reclaimed resources after eviction.
 		"--v", LOG_VERBOSITY_LEVEL, "--logtostderr",
+		"--fail-swap-on=false",
```

Comment:
I would prefer not setting this to false by default.  Running with swap can produce some unusual behavior in k8s, and I would prefer that people have to override this specifically.

Reply:
Thanks, addressed your review, PTAL.